recipes/avahi/all/conanfile.py

```diff
@@ -41,20 +48,14 @@ def requirements(self):
         self.requires("gdbm/1.19")
         self.requires("libevent/2.1.12")
 
-    def build_requirements(self):
-        self.tool_requires("glib/2.75.2")
-        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
-            self.tool_requires("pkgconf/1.9.3")
-
     def validate(self):
         if self.settings.os != "Linux":
             raise ConanInvalidConfiguration(f"{self.ref} only supports Linux.")
 
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
-        self.settings.rm_safe("compiler.cppstd")
-        self.settings.rm_safe("compiler.libcxx")
+    def build_requirements(self):
+        self.tool_requires("glib/2.75.2")
```

Comment:
Can you bump the glib version while you're at it? Both here and in the `requirements` above?

```suggestion
        self.tool_requires("glib/2.76.3")
```

Reply:
The removal of test_v1_package has been requested in most previous PRs, so I thought it's safe to simply include it in all new PRs.
The CMake properties were intended for the newer generators' support, but on second thought these are likely unnecessary if their default value is the component's name.

Reply:
The pkg_config_name property replaces the v1 way of setting the same value. It should be compatible with v1.40+ or so versions.

Reply:
We should probably drop them if the defaults work. I know that since Avahi doesn't provide native CMake Find modules, we're supposed to leave that as the default provided by the generator. Since the defaults for PkgConfig are fine, we can probably leave those off, too.